Rules/AvoidUsingInternalURLs.cs

```diff
@@ -118,7 +113,12 @@ public IEnumerable<DiagnosticRecord> AnalyzeScript(Ast ast, string fileName)
                             }
                             if (!firstPartURL.Contains("."))
                             {
-                                isInternalURL = true;
+                                //Add a check to exclude potential SDDL format. Check if a string have four components separated by ":"
```

Comment:
Are we checking if the SDDL formats are correct as well? This check is just to exclude SDDL strings being warned as internal URLS.